development/plugins-build-plugin.rst

```diff
@@ -0,0 +1,203 @@
+Files and directory
```

Comment:
Wouldn't it make sense to move that into under `Development > Plugin Development`?
A short introduction that describes what this chapter is about would be helpful for the users to understand what they are watching here.